automan/cluster_manager.py

```diff
@@ -326,6 +326,15 @@ def _update_sources(self, host, home):
         )
         self._run_command(cmd)
 
+    def _delete_outputs(self, host, home, sim_dir):
+        path = os.path.join(home, self.root, self.project_name,
+                            sim_dir)
+        real_host = '' if self.testing else '{host}'.format(host=host)
```

Comment:
This will wipe out local files in the testing case?  Is that what you intend?

Reply:
Yes, It skips localhost and the worker having nfs=True.